docs/photoniq/p3/view-p3-alerts.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 60
+title: View P3 Alerts
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+After you have created one or more policies, you can track P3 metrics on the dashboard.
+
+## View Alerts
+
+1. Log in to your PhotonIQ Performance Proxy dashboard.
+2. Click **Alerts & Audits**.
+3. (Optional) In the Alerts table, click **ACK** to acknowledge the alert.
+
+![View P3 Alerts](/img/photoniq/p3/p3-view-alerts.png)
+
+## Alert Fields
+
+The following fields list information about the alert:
+
+- **URL** - The URL that the warning or error applies to.
+- **Status** - Current status. Possible statuses are: Failed, XXX, or XXX.
+- **Timestamp** - The time when the warning or error was logged.
+- **Policy Name** - The name of the policy that the warning or error is associated with.
+- **Action** - Possible actions for you to take. You can click **ACK** to acknowledge the warning or error, or ...
```

Comment:
What actions are possible, what does it mean if the user performs them, such as clicking ACK?